cluster-provision/gocli/opts/cnao/manifests/network-addons-config-example.cr.yaml

```diff
@@ -9,3 +9,5 @@ spec:
   kubeSecondaryDNS: {}
   linuxBridge: {}
   macvtap: {}
+  multus: {}
```

Comment:
This change belong to previous commit.

Reply:
if you are referring to the multus issue then it was squashed